_posts/2023-04-11-mid-level-swe-resume.md

```diff
@@ -0,0 +1,196 @@
+statement. But where the objective statement is only one sentence that likely
+doesn't make the case for your interest, the cover letter makes up for that. It
+gives you the room to make the case for why your skills are transferrable, and
+let's you talk about what it is that interests you in that realm of technology.
+I reccomend having a cover letter ready for almost every job you apply for.
+Once you have one it is a relatively low effort task to adjust it for a given
+position and it can help set your application apart from someone who has the
+same experience, but no letter.
+
+# Other info and closing thoughts:
+&nbsp;&nbsp;&nbsp; The resume is the first step in your job hunt and it pays to
+get it right. Have several people review it. Have your teammates review it, an
+engineering manager (for obvious reasons probably not your current manager), a
+professional editor, and an engineer on the team you are applying for if you
+can manage that. Once you have your resume keep a version saved for every
+position you apply to (so the resume you are looking at is consistent with what
+the interviewer has in hand), then keep updating a base copy every few months
+while your experience is fresh in your mind. After you have a good resume write
+a cover letter. You may not use it with every application but it's useful to
+have ready for the ones that you really care about.
+
+&nbsp;&nbsp;&nbsp; Once all the written material is out of the way start
+applying. I would start with a job that you don't care about and know that you
+won't take. Getting an interview out of the way can be a big boost to your
+confidence or show you what you need to improve most on. Also, apply to Google.
+Even if you don't want to work there, it will likely be the most difficult
+coding interview that you have, so preparing for that will help with every
+other application. Then, I would start applying to one position a night, for
+about two weeks. With 14 active applications your chances of getting a few
+calls are not bad (for me the hit rate was just over 40%). While doing all this
+you probably want to start working on your code interview skills.
+
+&nbsp;&nbsp;&nbsp; This last though is a gamble, but worth considering.
+Depending on how serious you are about finding a new position, it may be worth
+leaving your current job. There are a **lot** of factors here to consider, but
+preparing for interviews is almost like a full time job by itself. Often
+interviews gauge a completely different skillset then you have in your current
+position (or stupidly enough, the position you are applying for), so taking the
+time to solely focus on being ready for that can pay off. Even if you do get an
+offer, if you join already burnt out when you start that can also be very
+tough. Regardless of what you choose to do, have a plan. Interviewing can
+easily burn you out mentally, physically, and emotionally, so be prepared.
+
```

Comment:
Add a little bit about sharing your resume on LinkedIn (be sure to remove phone number)